docs/getting_started/astro.rst

```diff
@@ -5,13 +5,18 @@ Getting Started on Astro
 
 While it is possible to use Cosmos on Astro with all :ref:`Execution Modes <execution-modes>`, we recommend using the ``local`` execution mode. It's the simplest to set up and use.
 
+If you'd like to see a fully functional project to run in Astro (CLI or Cloud), `check out cosmos-demo <https://github.com/astronomer/cosmos-demo>`_.
+
+Below you can find the step-by-step to run your own dbt project within Astro.
+
 Pre-requisites
 ~~~~~~~~~~~~~~
 
 To get started, you should have:
 
 - The Astro CLI installed. You can find installation instructions `here <https://docs.astronomer.io/astro/cli/install-cli>`_.
 - An Astro CLI project. You can initialize a new project with ``astro dev init``.
+- A dbt project. The `jaffle shop example <https://github.com/dbt-labs/jaffle_shop>`_ is a good example.
```

Comment:
Looks like navigating to this GitHub project is getting redirected to https://github.com/dbt-labs/jaffle-shop-classic, but I guess this should be fine

Reply:
Thanks, @pankajkoti , I just updated to the new URL!